### DISCUSSION TITLE
【委托】委托奖励：联动款奶茶

### DISCUSSION THREAD

**Discussion by kira-1-kira** [share121/inter-knot#3430] [其他]
委托任务：帮我去空洞内找400丁尼，400=328+22（奶茶）+50，别问我328和50是什么意思，我明天自有安排<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by 1chidai** [comment]
这个委托还是算了吧
